.flowconfig

```diff
@@ -3,7 +3,6 @@
 .*/node_modules/fbjs/*
 .*/node_modules/jss/lib/.*\.js\.flow
 .*/node_modules/react-swipeable-views/src/.*
-.*/node_modules/react-native/*
```

Comment:
react-native is not longer a dependency of `react-swipeable-views`.
I have broken that package into 4 with *lerna*.